include/deal.II/lac/block_indices.h

```diff
@@ -285,6 +309,21 @@ BlockIndices::BlockIndices (const std::vector<size_type> &block_sizes)
 }
 
 
+
+#ifdef DEAL_II_WITH_CXX11
+inline
```

Comment:
Make it a bit easier to read by inserting an empty line between the `#ifdef` and `inline`. Same after the function.